doc/source/background.md

```diff
@@ -0,0 +1,59 @@
+(background)=
+
+# Background
+
+More information on the people behind GeoUtils, and the package's mission.
+
+## Mission
+
+```{epigraph}
+The core mission of GeoUtils is to be **easy-of-use**, **robust**, **reproducible** and **fully open**.
+
+Additionally, GeoUtils aims to be **efficient**, **scalable** and **state-of-the-art**.
+```
+
+In details, those mean:
+
+- **Ease-of-use:** all basic operations or methods only require a few lines of code to be performed;
+
+- **Robustness:** all methods are tested within our continuous integration test-suite, to enforce that they always perform as expected;
+
+- **Reproducibility:** all code is version-controlled and release-based, to ensure consistency of dependent packages and works;
+
+- **Open-source:** all code is accessible and re-usable to anyone in the community, for transparency and open governance.
+
+```{note}
+:class: margin
+Additional mission points, in particular **scalability**, are partly developed but not a priority until our first long-term release ``v0.1`` is reached.
+```
+
+And, additionally:
+
+- **Efficiency**: all methods should be optimized at the lower-level, to function with the highest performance offered by Python packages;
+
+- **Scalability**: all methods should support both lazy processing and distributed parallelized processing, to work with high-resolution data on local machines as well as on HPCs;
+
+- **State-of-the-art**: all methods should be at the cutting edge of remote sensing science, to provide users with the most reliable and up-to-date tools.
+
+
+## The people behind GeoUtils
+
+```{margin}
+<sup>2</sup>More on our GlacioHack founder at [adehecq.github.io](https://adehecq.github.io/)!
+```
+
+GeoUtils was created during the [GlacioHack](https://github.com/GlacioHack) hackaton event, that was initiated by
+Amaury Dehecq<sup>2</sup> and took place online on November 8, 2020.
+
+```{margin}
+<sup>3</sup>Check-out [glaciology.ch](https://glaciology.ch) on our founding group of VAW glaciology!
+```
+
+The initial core development of GeoUtils was performed by members of the Glaciology group of the Laboratory of Hydraulics, Hydrology and
```

Comment:
This scope of contributors and 'founding history' is a bit narrow at the moment. Either expand it, or drop this section completely (this would be my preference, as GitHub provides a good summary of the contributors already).